src/c++/perf_analyzer/schema.json

```diff
@@ -0,0 +1,96 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/triton-inference-server/client/tree/main/src/c%2B%2B/perf_analyzer/output",
+    "title": "Perf Analyzer output data",
+    "description": "A json file describing the output from a perf analyzer run",
+    "type": "object",
+    "properties": {
+        "experiments": {
+            "description": "The array of all experiments run by perf analyzer",
+            "type": "array",
+            "items": {
+                "type": "object"
+            },
+            "minItems": 1,
+            "uniqueItems": true,
+            "properties": {
+                "experiment": {
+                    "description": "A single experiment run by perf analyzer",
+                    "type": "object",
+                    "minItems": 1,
+                    "maxItems": 1,
+                    "properties": {
+                        "mode": {
+                            "description": "Operating mode of perf analyzer: either concurrency or request rate",
+                            "type": "string"
+                        },
+                        "value": {
+                            "description": "Concurrency or request rate for the current experiment",
+                            "type": "integer"
+                        }
+                    },
+                    "required": [
+                        "mode",
+                        "value"
+                    ]
+                },
+                "requests": {
+                    "description": "The array of requests sent by perf_analyzer for this experiment",
+                    "type": "array",
+                    "uniqueItems": true,
+                    "items": {
+                        "$ref": "#/properties/experiments/properties/$defs/request"
+                    }
+                },
+                "$defs": {
+                    "request": {
+                        "description": "Info for a single request",
+                        "type": "object",
+                        "properties": {
+                            "timestamp": {
+                                "description": "Time stamp of the request",
+                                "type": "integer"
+                            },
+                            "sequence_id": {
+                                "description": "sequence_id of the request",
+                                "type": "integer"
+                            },
+                            "responses_timestamps": {
+                                "description": "All associated responses to this request",
+                                "type": "array",
+                                "items": {
+                                    "type": "integer"
+                                }
+                            }
+                        },
+                        "required": [
```

Comment:
Just a quick check: Should `sequence_id` also be required as part of request?

Reply:
non-sequence models will not have a sequence ID